workflow/build_opts.yaml

```
@@ -0,0 +1,90 @@
+systems:
+  common:
+    - "ufs_utils"
+    - "gfs_utils"
+    - "upp"
+  gfs:
+    - "gfs_model"
+    - "gfs_ww3prepost"
+    - "gsi_enkf"
+    - "gdas"
+    - "gsi_utils"
+    - "gsi_monitor"
+  gefs:
+    - "gefs_model"
+    - "gefs_ww3_prepost"
+  sfs:
+    - "sfs_model"
+    - "gefs_ww3_prepost"
+build:
+  gfs_model:
+    command: "./build_ufs.sh -e gfs_model.x -j 12"
+    log: "build_ufs_gfs.log"
+    cores: 12
+    walltime: "00:30:00"
+
+  gfs_ww3prepost:
+    command: "./build_ww3prepost.sh -j 4"
+    log: "build_ww3prepost_gfs.log"
+    cores: 4
+    walltime: "00:10:00"
+
+  gefs_model:
+    command: "./build_ufs.sh -w -e gefs_model.x -j 12"
+    log: "build_ufs_gefs.log"
+    cores: 12
+    walltime: "00:30:00"
+
+  gefs_ww3_prepost:
+    command: "./build_ww3prepost.sh -w -j 4"
+    log: "build_ww3prepost_gefs.log"
+    cores: 4
+    walltime: "00:10:00"
+
+  sfs_model:
+    command: "./build_ufs.sh -e sfs_model.x -j 12"
```

```suggestion
    command: "./build_ufs.sh -y -e sfs_model.x -j 12"
```
I think the SFS model is built with a hydrostatic option.

---

Yep, fixed.